### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Prishruit**
Added wandb link for benchmark of Atari.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ To install Marlgrid, run
 - [DQN](https://github.com/chandar-lab/RLHive/blob/main/hive/agents/dqn.py)
 - [Full Rainbow Agent](https://github.com/chandar-lab/RLHive/blob/main/hive/agents/rainbow.py)
 
+## Atari Benchmark
+Results of our implimentations of DQN and Rainbow on Atari games Breakout, Pong, Space Invaders, Seaquest and Qbert over 10 seeds can be accessed here - [Wandb RLhive Atari](https://wandb.ai/chandar-rl/Hive-baselines/reports/Atari--VmlldzoyMjMyODMw) 
+
```
Comment (dapatil211) [inline README.md]: We don't want just the atari benchmark on here. We want a link to the main benchmark report which links to benchmarks for all environments

**Review by dapatil211 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Prishruit) [reply README.md]: done, Updated

File: README.md [hunk 2]
```diff
@@ -31,6 +31,9 @@ To install Marlgrid, run
 - [DQN](https://github.com/chandar-lab/RLHive/blob/main/hive/agents/dqn.py)
 - [Full Rainbow Agent](https://github.com/chandar-lab/RLHive/blob/main/hive/agents/rainbow.py)
 
+## Benchmark
+Reproduced results of our implimentations are available here - [Wandb RLhive](https://wandb.ai/chandar-rl/Hive-baselines/reports/Hive-Baselines--VmlldzoyMjMyODE2), this includes DQN and Rainbow models on Atari games and TD3 and DDPG on MuJoCo.
```
Comment (dapatil211) [inline README.md]: Just do the line:
We reproduce baselines for our agents [here](https://wandb.ai/chandar-rl/Hive-baselines/reports/Hive-Baselines--VmlldzoyMjMyODE2)!

Also move this to the line below "The full documentation and tutorials are available at https://rlhive.readthedocs.io/."

**Review by dapatil211 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Prishruit) [reply README.md]: done

**Review by dapatil211 — APPROVED** [review]

**Review by hnekoeiq — APPROVED** [review]

**Pull request merged**